README.md

```diff
@@ -271,7 +271,7 @@ Kafka Streams examples via:
 #
 $ mvn clean package
 
-# >>> Creates target/kafka-streams-examples-5.0.0-SNAPSHOT-standalone.jar
+# >>> Creates target/kafka-streams-examples-5.1.0-SNAPSHOT-standalone.jar
```

Comment:
Remove the `SNAPSHOT` here. Ditto elsewhere

Reply:
For `5.1.x` we will keep the snapshot, for `5.1.y-post` we will remove the snapshot suffix.

Reply:
Ah, yeah that makes sense.